security-response.md

```diff
@@ -0,0 +1,107 @@
+# Security Response Guidelines
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable to
+publicly known exploits.
+
+The OpenTelemetry (OTel) Technical Committee (TC) and relevant repo maintainers,
+supported by tooling provided by the Sig-Security, are responsible for
+responding to the incident organizing the entire response including internal
+communication and external disclosure.
+
+## Supported Versions
+
+The OTel project provides community support only for the last minor
```

Comment:
1. what happens for problems affecting only older versions? are users required to migrate to the latest?
2. is this a current policy, or is it new? if it's new, we should bring this to the TC/maintainers

Reply:
This is directly pulled from our default security doc in GitHub but I imagine no one has really thought about it in awhile